docs/self-managed/components/orchestration-cluster/core-settings/configuration/webserver.md

```diff
@@ -0,0 +1,51 @@
+---
+id: webserver
+title: "Webserver & security"
+description: "This page covers the configuration settings related to the webserver and security components of the Camunda 8 Orchestration Cluster."
+---
+
+## Operate
+
+Operate supports customizing the **context-path** using default Spring configuration.
+
+Example for `application.yml`:
+`server.servlet.context-path: /operate`
+
+Example for environment variable:
+`SERVER_SERVLET_CONTEXT_PATH=/operate`
+
+The default context-path is `/`.
+
+### Security
```

Comment:
This section is obsolete not correct anymore. https://github.com/camunda/camunda-docs/pull/5969 covers the same topic with the proper 8.8 configuration. So you could consider removing the content here and/or update it once the linked PR is merged.

Reply:
Removed this 👍